frontend/src/scenes/session-recordings/playlist/sessionRecordingsPlaylistLogic.test.ts

```diff
@@ -115,6 +116,7 @@ describe('sessionRecordingsPlaylistLogic', () => {
             },
         })
         initKeaTests()
+        featureFlagLogic.mount()
     })
```

Comment:
Do we have an example for how to mock feature flag logic in tests (to set a value, and make sure the UI behaves as expected)

Reply:
no, this comes up periodically 🙈 

i think the best way is to mock the response to initial teams API call, which will populate the flag defaults iirc